util/paywall.go

```diff
@@ -458,73 +381,35 @@ func convertBEPrimaryTransactionToTxDetails(address string, tx BEPrimaryTransact
 	}, nil
 }
 
-func convertBEBackupTransactionToTxDetails(tx BEBackupTransaction) (*TxDetails, error) {
-	amount, err := DcrStringToAmount(tx.Amount.String())
-	if err != nil {
-		return nil, err
-	}
-
-	return &TxDetails{
-		Address:       tx.Address,
-		TxID:          tx.TxId,
-		Amount:        amount,
-		Confirmations: tx.Confirmations,
-		Timestamp:     tx.Timestamp,
-	}, nil
-}
-
 // FetchTxsForAddress fetches the transactions that have been sent to the
-// provided wallet address from the dcrdata block explorer. If the dcrdata
-// request fails the insight block explorer is tried.
+// provided wallet address from the dcrdata block explorer
 func FetchTxsForAddress(address string) ([]TxDetails, error) {
 	// Get block explorer URLs
```

Comment:
```suggestion
	// Get block explorer URL
```

No longer multiple URLs.